### DISCUSSION TITLE
[人工翻译] 将VPet翻译成日语

### DISCUSSION THREAD

**Discussion by unsolublesugar** [LorisYounger/VPet#197] [Translate]
你好！ 我是一名来自日本的宅男程序员。
感谢你们开发了一个非常可爱的桌面吉祥物。

我的一位朋友正在将 VPet 翻译成日语。
https://twitter.com/ggeek4/status/1691736670809554968?s=20

目前由志愿者分发的许多修改器都是粗制滥造的机器翻译，有损 VPet 的可爱性。
我的朋友还与台湾的朋友有联系，我们能够准确、可爱地翻译日语。

我们还在翻译 VPet 的过程中，但如果您愿意，如果您愿意，我很乐意看到它被正式纳入。
一旦获得批准，我们将着手开展全面的翻译工作，Pull Request 我想继续创建一个

我们将非常感谢您的考虑！

# 参考图像
![2023-08-17_14h07_41](https://github.com/LorisYounger/VPet/assets/8685879/10d589b4-be99-462e-8069-0c819b87b618)
![2023-08-17_14h06_26](https://github.com/LorisYounger/VPet/assets/8685879/96a1ebd4-d058-4321-84f1-095fd5a8cdb3)
![2023-08-17_14h05_51](https://github.com/LorisYounger/VPet/assets/8685879/c7d94bdb-67c3-449f-88d6-6ed6a4bf58f5)
![2023-08-17_14h04_44](https://github.com/LorisYounger/VPet/assets/8685879/66940a11-1b7a-43ef-9aa6-d5cf41934fd6)

## 翻译的一部分
```
好无聊呀#もう、つまらない～！:|
干点什么好呢#何かしようかな？:|
主人~在干嘛呢#ご主人様~なにしてるの？:|
想你啦主人~#ご主人様、会いたいよ～。:|
陪我玩嘛主人~#ご主人様、かまってよ～！:|
主人~主人~主人~#ご主人様～ご主人様～ご主人様！:|
主人抱抱~#ハグして～！:|
阿姨洗铁路~主人~#ご主人様、大好き！！:|
我已经离不开主人了呢~#もうご主人様なしではだめだよ～。:|
其实我知道主人和我不在一个世界里，但是，我对主人的爱是不会变的！我一定会*到主人身边#！等着我#主人±-~锟斤拷烫烫烫**^^&%$**#実は、ご主人様と私、違う世界にいるの知ってるけど、主人への愛は変わらないよ！絶対にご主人のところに行くからね!待っててね#ご主人ｑ±-╠╠╠╠╠╠╠╠^^&%════════$:|:|
嗯哼哼~#うーむ。:|
怎么了吗？#どうしたの?:|
需要我做什么呢？#なにか手伝ってほしいの？:|
在哪有睡午觉的好地方呢#お昼寝する良い場所、どこかな？?:|
让我康康主人的学习资料里都有什么#ご主人様の宿題、ちょびっと見せて～？:|
走呀走呀走#歩こう～♪歩こう～♪:|
我才不是闲人呢！#暇人じゃないもん！:|
好想找点事情做呀#何かしたいなぁ？:|
好闲哦！好无聊！#暇すぎ！つまらないよ！:|
闲的我好困啊#もうだらけて眠いな～。:|
闲到难受。。#ひまで辛いです..。:|
好好学习天天向上#毎日、一生懸命に勉強するよ！:|
```

**Comment by LorisYounger** [comment]
感谢你们的支持, 不过因为目前创意工坊里有非常多版本的翻译文档, 我无法抉择让某一个成为官方翻译
你可以在标题中添加 [人工翻译] ，让大家知道你们翻译的优势
我将会把这个issue保持开启,这样方便你们互相联系翻译人员
非常感谢你对这个游戏的支持

Thank you for your support, but because there are currently many versions of translation documents in the workshop, I cannot choose to make one of them an official translator.
You can add [Human Translation] in title that Let everyone know the advantages of your translation
I will keep this issue open so that you can communicate with the translators
Thank you very much for your support of my game

**Comment by MHYCWasTaken** [comment]
使用crowdin翻译社区也许可以解决翻译版本问题  
crowdin.com

**Comment by LorisYounger** [comment]
> 使用crowdin翻译社区也许可以解决翻译版本问题 crowdin.com

谢谢,我去看看

**Comment by unsolublesugar** [comment]
@LorisYounger 
感谢您在百忙之中抽空回复！

确实，现在有很多翻译文件，很难判断
日语翻译回复完成后，我们将 ，我会提交一个 Pull Request

如果内容能得到其他用户的认同，我们希望您能考虑采纳

一旦有更新，我们会尽快通知您
我们将继续支持您的开发！
